docs/front-end/decision-log.md

```diff
@@ -0,0 +1,49 @@
+# Decision log
+
+## 1. Hosting of front-ends (FEs), backend-for-frontends (BFFs) and Design System in Green Force ART
+
+Date: 2021-09-20
+
+Decision: **Option A** has been chosen as **option B** can be applied later on, if necessary  
+
+Responsible: SSG, MEA, XALEH, XDZUS
```

Comment:
Please use our GitHub handles:
``` suggestion
Responsible: @Sondergaard, @MartinFHansen, @frontend-specialisten, @dzhavat
```

Reply:
Thanks for the clarification. Makes sense when we're on GitHub.